src/HookFactory.php

```diff
@@ -129,6 +131,12 @@ public function generateHook($type, $programPath, $programName = null)
         // Use the program path if an alias/name is not given
         $programName = $programName ?: $programPath;
 
+        if ($multiple) {
+            $completionCommand = '${1} _completion';
```

Comment:
That's the first thing I've checked before wanting to to ask you why you've merged this PR and checked the merge result to find the error instead tell me to fix that and merging only afterwards.